exampleSite/config.toml

```diff
@@ -49,8 +49,17 @@ disqusShortname = "yourdiscussshortname"
     # Enable Twemoji
     enableTwemoji = true
 
+    # Remote font and css
+    use_default_remote_font = true
```

Comment:
In general, Hugo parameters do not have underscores in their names (like `enableTwemoji` instead of `enable_twemoji`. There isn't a style guide, but it is common use.